db/seeds.rb

```diff
@@ -1,35 +1,105 @@
+Beer.find_or_create_by!(name: "Old Rasputin", user: user, brewery: Brewery.find_by(name: "North Coast Brewing Company"), style: Style.find_by(name: "Imperial Stout"), abv: 9, ibu: 75, description: "A rich, intense brew with a robust palate, a fruity nose and a warming finish. Very complex.")
+Beer.find_or_create_by!(name: "Duvel", user: user, brewery: Brewery.find_by(name: "Brouwerij Duvel Moortgat"), style: Style.find_by(name: "Belgian Strong Ale"), abv: 8.5, description: "Four generations of the Moortgat family have brought together Pilsener malts, Bohemian hops, and a unique yeast strain to create this intense, aromatic and beguiling ale.")
+Beer.find_or_create_by!(name: "Weihenstephaner Hefeweissbier", user: user, brewery: Brewery.find_by(name: "Bayerische Staatsbrauerei Weihenstephan"), style: Style.find_by(name: "Hefeweizen"), abv: 5.4, ibu: 14, description: "Our golden-yellow wheat beer, with its fine-pored white foam, smells of cloves and impresses consumers with its refreshing banana flavour. It is full bodied and with a smooth yeast taste.")
+Beer.find_or_create_by!(name: "Samuel Adams Boston Lager", user: user, brewery: Brewery.find_by(name: "Samuel Adams (Boston Beer Company)"), style: Style.find_by(name: "Lager"), abv: 4.7, ibu: 30, description: "Samuel Adams Boston Lager® is the best example of the fundamental characteristics of a great beer, offering a full, rich flavor that is both balanced and complex.")
+Beer.find_or_create_by!(name: "Duck Duck Gooze", user: user, brewery: Brewery.find_by(name: "The Lost Abbey"), style: Style.find_by(name: "Wild Ale"), abv: 7, description: "As with all sour ales, the initial sip wakes up your dull senses. There is an elevated level of Acetic Acids working their magic in this beer. The first sip assaults the palate in an enamel stripping bottom of the pH chain sort of way.")
+Beer.find_or_create_by!(name: "Bigfoot", user: user, brewery: Brewery.find_by(name: "Sierra Nevada Brewing Company"), style: Style.find_by(name: "Barleywine"), abv: 9.6, ibu: 90, description: "Bigfoot is a beast of a beer, brimming with bold flavors of bittersweet malt and heaps of aggressive whole-cone Pacific Northwest hops.")
+Beer.find_or_create_by!(name: "80-Acre Hoppy Wheat Beer", user: user, brewery: Brewery.find_by(name: "Boulevard Brewing Company"), style: Style.find_by(name: "Wheat Beer"), abv: 5.5, ibu: 20, description: "With roots in two of today's most popular brewing styles, 80-Acre Hoppy Wheat Beer is the result of careful cultivation by our brewers and cellarmen. Their efforts to craft a hybrid yielded a delightfully distinctive ale with the aroma of an IPA and the refreshing taste of a wheat beer.")
```

Comment:
You could clean all this up by making a CSV file in a `db/data` directory and seed your database from there. Same thing with `Style`.